packages/host/app/modifiers/code-block.ts

```diff
@@ -142,6 +169,16 @@ export default class CodeBlock extends Modifier<Signature> {
   }
 }
 
+function makeCodeActionsContainerDiv() {
+  let template = document.createElement('template');
+  let randomId = Math.random().toString(36).substring(2, 15);
```

Comment:
Maybe we could use `guidFor(template)` instead?

Reply:
Yep, I'll use that instead.